meinberlin/apps/votes/tasks.py

```diff
@@ -7,7 +7,7 @@
 # Number of tokens to insert into database per bulk_create
 BATCH_SIZE = 1000000
```

Comment:
I believe also this has one zero too many, wasnt that 100.000 before @fuzzylogic2000 ?

Reply:
No, it used to be one million: the PAGE_SIZE in the exports: https://github.com/liqd/a4-meinberlin/commit/eacd22c795353a6a5a0ac9773f74dba6ae533bc5#diff-ef2af1f23503a2f006e8048ad40751390211bb74ca18c6882369e290b66f022bL10
And yeah, we could use one variable as they are the same size, but on the other hand, they are for different things, so if we ever want to change only one (like if we change the background tasks somehow) it does make sense to keep them seperated. So, I think, we should keep two numbers.

Reply:
@Rineee

Reply:
Right, @Rineee was right! The BATCH_SIZE is the other number... That is also wrong and should be one zero shorter.